.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: node_js
+node_js:
+  "12.4.0"
```

Comment:
I see that the LTS version ("recommended for most users") is 10.16.0. Should we back off from the bleeding edge?

Reply:
Hmm, this is a good question.  I don't remember why I went with 12.4.0.  I can try bumping down to 10.16.0 and just make sure it still works ok.

Reply:
Seems to still work OK, so I'll use 10.16.0 instead.